src/config.rs

```diff
@@ -315,8 +322,8 @@ mod config_test {
 
     #[test]
     fn test_default_config_path() {
-        let expected = dirs::config_dir().unwrap().join("pike.toml");
-        let actual = super::default_config_path().expect("Failed to get default config path");
+        let expected = dirs::config_dir().unwrap().join("pike").join("pike.toml");
+        let actual = super::default_config_file_path().expect("Failed to get default config path");
```

Comment:
tu mozna by zamiast robic super:: to zaimportowac bo to nasz kod ktory wolamy, ja sam sie zaczynam zastanawiac czy wolajac cudzy kod dobrze wyglada jak sie rzuca tymi modulami tak zamiast robic dokladne importy

Reply:
pardon, nie wylapalem wczesniej